2-mandatory/1-weather-report.js

```diff
@@ -11,8 +11,14 @@
         - Hint: you can call the temperatureService function from your function
 */
 
+let temp;
 function getTemperatureReport(cities) {
-    // TODO
+    strings = [];
+    cities.forEach(item => {
+        temp = temperatureService(item);
+        strings.unshift(`The temperature in ${item} is ${temp} degrees`);
```

Comment:
On line 16 the variable string is not a meaningful variable name and it hasn't been declared nor initialised.
on line 17 the item argument should be replaced by city so that it will be easier for the reader to understand it.   
On line 19 we should use the push method instead of unshift method and the item variable should be replaced by city.

Reply:
This is great feedback, you've said both what is wrong and why it matters 👍